index.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Tick Tac Toe</title>
+  <script src="./script.js" async defer></script>
+  <link rel="stylesheet" href="./styles.css">
+</head>
+<body>
+    <section>
+        <h1 class="gameName">Tick Tack Toe</h1>
+        <div class="playerTurn">Player turn: x</div>
+        <div class="scoreBoard"></div>
+        <div class="container">
+            <div class="gameCells" id="0"></div>
```

Comment:
iirc, classes and id's can't start with a numeric character. This is confusing cuz sometimes js is just ok with it and sometimes it actually complains. It's good that you got it working, but in general, a good trick is to give them ids like "box-1", and then when working with them you can .replace('id-', '') to snip it off and just get the id